task_sql/notes.txt

```diff
@@ -1,3 +1,20 @@
-From docker command line:
+When creating your .env file (python script):
+  Leave the username and password as the defaults: 'postgres'
+  Set the database name as: 'wondertix'
+
+
+Connecting PGAdmin to the DB in docker:
+  Download PGAdmin: https://www.pgadmin.org/download/
+  Start the docker project
+  Open PGAdmin and select 'Add New Server'
+    Under the 'General' tab, give the server a name
+    Under the 'Connection' tab:
+      Set hostname/address to 'localhost'
+      Verify Port is set to 5432
+      Set username/password to the same values in your .env file (postgres default for both)
+  Click save and you will be connected
+
+
+To connect to the DB from docker command line:
```

Comment:
Docker command line is a little ambiguous, I would suggest elaborating here more e.g. "From Docker Desktop, go to the server container and select 'container actions' -> 'open in terminal'" or something like that